python/common/org/python/types/Str.java

```diff
@@ -708,10 +708,21 @@ private long toPositiveIndex(long index) {
     }
 
     @org.python.Method(
-            __doc__ = ""
+            __doc__ = "string.isalnum() -> bool\n" +
+                    "\n" +
+                    "Returns false when the string has atleast ONE character which is\n" +
```

Comment:
Ah, I see. We don't currently have a way of automatically verifying methods' docstrings against what Python has, but you should use the actual ones where possible! For this particular method you can get it by `str.isalnum.__doc__` in a Python console.

(Please do this for isdecimal below as well.)